src/amp.js

```diff
@@ -64,7 +64,9 @@ import {stubElementsForDoc} from './service/custom-element-registry';
  * @type {boolean|undefined}
  */
 const shouldMainBootstrapRun = !self.IS_AMP_ALT;
-
+// NO SUBMIT PLZ
```

Comment:
This change was to trigger build targets, "DO NOT SUBMIT" will make Travis fail before running the tests themselves